src/test/java/nextstep/ladder/LadderTest.java

```diff
@@ -8,16 +8,39 @@
 import org.junit.jupiter.api.Test;
 
 import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
 
+import static java.util.Arrays.asList;
 import static org.assertj.core.api.Assertions.assertThat;
 
 public class LadderTest {
 
     @Test
     @DisplayName("사다리는 여러개의 선으로 이루어진다")
     void 사다리_생성() {
-        Line line = new Line(new Point(Direction.LEFT));
+        Line line = new Line(new Point(Direction.STRAIGHT));
         Ladder ladder = new Ladder(line);
         assertThat(ladder.ladder()).isEqualTo(Arrays.asList(line));
     }
+
+    @Test
+    @DisplayName("사다리를 움직이면 각 지점의 최종 위치가 나온다")
+    void 사다리_게임() {
+        Line line1 = new Line(asList(new Point(Direction.RIGHT,0), new Point(Direction.LEFT,1), new Point(Direction.RIGHT,2),new Point(Direction.LEFT,3)));
```

Comment:
```suggestion
        Line line1 = new Line(asList(Point.right(0), Point.left(1), Point.right(2), Point.left(3);
```

이와 같은 정적 메서드를 추가해 Point 생성을 좀 더 수월하도록 지원하는 것은 어떨까?

Reply:
훨씬 깔끔해 져서 좋을 것 같습니다. 하지만 고민이 되는 건 이전에 받은 피드백 중에 테스트를 위한 메서드를 지양 하는게 어떻겠냐는 의견이 있었습니다.  그래서 제안해주신 메서드가 사실 테스트를 위한 메서드라고 생각이 드는데 혹시 어떻게 생각하시나요?